x-pack/plugins/ml/public/application/explorer/_explorer.scss

```diff
@@ -219,9 +219,12 @@
     height: 30px;
     border-bottom: 0px;
     border-radius: 2px;
-    margin-top: -1px;
     white-space: nowrap;
 
+    &:not(:first-child) {
+      margin-top: -1px;
+    }
+
     div.lane-label {
       display: inline-block;
       font-size: 13px;
```

Comment:
Let's use `$euiFontSizeXS` instead.

Reply:
Changed in https://github.com/elastic/kibana/pull/70063/commits/081979542bae2b276991e2ccdfa8a36ab5573ac2